### PR TITLE
New Serverless pattern - EB Scheduler to EventBridge to Lambda - CDK Python

### PR DESCRIPTION
**Pull request by pubudusj**
*Issue #, if available:*

*Description of changes:*
Example added for EB Scheduler to EventBridge to Lambda with CDK Python.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eventbridge-schedule-to-eventbridge-cdk-python/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Delete stack
+
+```bash
+cdk destroy
+```
+
+---
+
+Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mavi888) [inline eventbridge-schedule-to-eventbridge-cdk-python/README.md:60]: 2023

File: eventbridge-schedule-to-eventbridge-cdk-python/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+6. Once the virtualenv is activated, you can install the required dependencies.
+    ```bash
+    $ pip install -r requirements.txt
+    ```
+7. To deploy the application:
+    ```bash
+    $ cdk deploy
+    ```
+
+## How it works
```
Comment (mavi888) [inline eventbridge-schedule-to-eventbridge-cdk-python/README.md:48]: add a section on how to test this pattern

**Comment by mavi888** [comment]
Thanks for your contribution. Please fix the comments below so i can merge this PR.

Cheers,

File: eventbridge-schedule-to-eventbridge-cdk-python/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+[EventBridge Scheduler](https://docs.aws.amazon.com/scheduler/latest/UserGuide/what-is-scheduler.html) is used to trigger a EventBridge event every 5 minutes.
```
[the remark this answers is not in the document]
Comment (pubudusj) [reply eventbridge-schedule-to-eventbridge-cdk-python/README.md]: This section was updated

File: eventbridge-schedule-to-eventbridge-cdk-python/README.md (hunk 1 above)
Comment (pubudusj) [reply eventbridge-schedule-to-eventbridge-cdk-python/README.md]: Fixed this.

File: eventbridge-schedule-to-eventbridge-cdk-python/README.md (hunk 2 above)
Comment (pubudusj) [reply eventbridge-schedule-to-eventbridge-cdk-python/README.md:48]: Testing section added.

**Comment by mavi888** [comment]
Thanks for sumitting your pattern.
Now this pattern is public in Serverless land

https://serverlessland.com/patterns/eventbridge-schedule-to-eventbridge-cdk-python

You can share your pattern using this link :)

**Pull request merged**